.travis.yml

```diff
@@ -51,6 +51,21 @@ matrix:
       env: ARCH=x86_64 LD=ld.lld-10 REPO=linux-next
       if: type = cron
     # stable
+    - name: "ARCH=arm32_v7 REPO=5.4"
```

Comment:
This should be able to be linked with `ld.lld`.

Reply:
done